### ISSUE TITLE
Json format doesn't work for multiple pages

### ISSUE THREAD

**Issue by hodgka** [rpiml/okcollege-web-client#15]
If you try having a form.json with just 1 page, it works fine as can be seen here:

![image](https://cloud.githubusercontent.com/assets/8852535/19101391/f57e789c-8a95-11e6-97f3-a3db1fb25f22.png)

If you add more pages, it doesn't like dealing with a comma somewhere, as can be seen here:

![image](https://cloud.githubusercontent.com/assets/8852535/19101351/99ea3c0a-8a95-11e6-9be7-54e0202e1a4a.png)

I'm assuming it's the comma in between one page, and the next i.e.

``` ... "next": "scores"
    }``` **,**```{
        "id": "scores",
        "questions": ... ```

**Comment by hodgka** [comment]
[Link to form.json in question.](https://github.com/rpiml/okcollege-dev/tree/form_json/assets)

Shouldn't be a syntax error in the JSON since Atom sorta lints json files. I'm assuming it has to do with how the json is getting parsed when it is pulled in by the FormComponent/surveyPageReducer.

Will fix tomorrow(maybe).

**Comment by seveibar** [comment]
Thanks for making the issue Alec, I'll take a look tonight

**Comment by jowlee** [comment]
This has been resolved in #16
